src/SFML/Audio/SoundStream.cpp

```diff
@@ -35,9 +35,11 @@ sfSoundStream* sfSoundStream_create(sfSoundStreamGetDataCallback onGetData,
                                     sfSoundStreamSeekCallback    onSeek,
                                     unsigned int                 channelCount,
                                     unsigned int                 sampleRate,
-                                    void*                        userData)
+                                    void*                        userData,
+                                    sfSoundChannel*              channelMapData,
+                                    size_t                       channelMapSize)
```

Comment:
Corresponding declaration was not updated.
Also I am wondering if maybe `userData` should remain the last parameter.

Reply:
Why should `userData` remain the last parameter?

Reply:
Sounds good. I'll make that change.